ArduPlane/tiltrotor.cpp

```diff
@@ -489,6 +494,9 @@ void Tiltrotor::tilt_compensate_angle(float *thrust, uint8_t num_motors, float n
             thrust[i] *= scale;
         }
     }
+    if (!fully_fwd()) {
+        SRV_Channels::set_output_scaled(SRV_Channel::k_throttle_tilt, largest_tilted*100);
+    }
```

Comment:
I guess this is to make sure the slew starts off in the right place? I would prefer to see it done with the throttle output value from motors. You should be able to use motors thrust linearisation like we do for tailsitter here:

https://github.com/ArduPilot/ardupilot/blob/5f8a655f35a2a7901a56e898609a2d7c488a99dd/ArduPlane/tailsitter.cpp#L321-L323